tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -1674,16 +1675,16 @@ public function getJsonDecodedResponseBodyContent(ResponseInterface $response =
 	}
 
 	/**
-	 * @Then the JSON data of the response should match
+	 * @Then the ocs JSON data of the response should match
 	 *
-	 * @param PyStringNode $schemaString
+	 * @param PyStringNode|string $schemaString
```

Comment:
step will always send PyString, isn't it? :thinking: 
```suggestion
	 * @param PyStringNode $schemaString
```